static/js/settings_account.js

```diff
@@ -575,14 +576,19 @@ exports.set_up = function () {
             processData: false,
             contentType: false,
             success: function () {
-                loading.destroy_indicator($("#upload_avatar_spinner"));
-                $("#user_avatar_delete_button").show();
+                $('#user-avatar-background').css({display: 'none'});
+                $('#user-avatar-spinner').css({display: 'none'});
+                $('#user_avatar_upload_button').show();
+                $('#user_avatar_delete_button').show();
```

Comment:
Can you refactor to move these blocks of common logic into two functions declared above?  E.g. here would be `display_avatar_upload_complete()` and the block above would be `display_avatar_upload_started()`

We might be able to move the couple lines below into those functions as well.